### ISSUE TITLE
User prlimit API to restrict maximum number of open files

### ISSUE THREAD

**Issue by aams574** [ME-CLASS/ME-2020#2]
Modify the following program to restrict maximum number of files that a process can open. 
Assume name of the program is flimit.  You can run the program like below:

1.flimit 2 -> This will set the maximum number of files to be opened to only 2.
1. flimit 5 -> This will set the maximum number of files to be opened to only 5.
2. flimit -1 -> This will set the maximum number of files to open to maximum limit.

Basically, you are passing the limit as command line argument. (similar to the prime numbers program)

If the program try to open more files than the limit, show that file pointer will be NULL.

https://github.com/PacktPublishing/Hands-on-System-Programming-with-Linux/blob/master/ch3/rlimit_primes.c
